server/lib/routes/get-index.js

```diff
@@ -2,6 +2,20 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
+var activityEvent = require('../activity-event');
+var crypto = require('crypto');
+
+var activityEventQueryParams = [
+  'context',
+  'entrypoint',
```

Comment:
It's certainly a good idea to track client_id here somehow.

Do we ever have both `service` and `client_id` on a single request?  I feel like there's prior art for treating them as aliases, e.g. in some existing metric slike the "account creates by service" graph we treat them interchangeably.  So perhaps it makes sense to pass the `client_id` as the value of `service` here.